modules/ROOT/pages/about-my-applications.adoc

```diff
@@ -12,14 +12,30 @@ image::ex2-my-applications.png[]
 A contract holds information about the contract with a client ID and client secret for the API.
 Each application lists a description, its URL, redirect URIs, grant types, and a usage dashboard.
 
-The usage information lets you set the verification duration. If not specified, the range is the last 8 days. The usage lists the total number of requests to the API, average latency, the error rate, and graphs for total requests, average latency, requests by HTTP status code, and the error percentage.
-
-The applications result when users click Request Access to establish
-a link between an application and an API. Either the application already exists,
-or the user creates the application from the Request Access menu.
+The usage information enables you to set the verification duration. If not specified,
+the range is the previous eight days. The usage lists the total number of requests to
+the API, average latency, the error rate, and graphs for total requests,
+requests by HTTP status code, and the error percentage.
 
 Applications in My Applications are registered using API Manager in Anypoint Platform.
 
+To implement an API, the client application must first request access. When the
+request is approved by the API, a contract is created between the client
+application and the API, and the client application is registered.
+
+If you don't have an application to link to your API, you can create the application
+in the Request Access menu. You can then click Request Access to connect your
+application and the API.
+
+APIs that are protected by a client ID enforcement policy require client
+applications to provide a client ID and client secret. The client ID and client
+secret credentials are automatically created when the client application is
+registered. These credentials enable the application to access the API and
+provide consumer analytics for an API developer.
+
+API contracts are visible to API consumers in Exchange, and visible to API
```

Comment:
Not sure if this is true in general. If your are org owner, then yes. I believe (please double check) that only org owners and API owners have visibility over the contracts of a certain API.

Reply:
@nahuel-dallavecchia who can confirm this? Exchange developers?

Reply:
Try with Samuel Centurion, QA Lead of API Manager first.

Reply:
Thank you, Nahuel! Looks like Sebastian Lamelas on the Exchange team already answered this one: "The API owner has visibility over API contracts in API Manager (visibility of ALL contracts ever created for that API instance), whereas the API consumer has visibility only of the API contracts created between an API instance and *his* application in "My Applications" page in Exchange."

Reply:
Fixed.